app/components/admin/schools/cohort_component.rb

```diff
@@ -38,6 +56,34 @@ def build_relationships
           with_relationship_component(school:, school_cohort:, relationship:, superuser:)
         end
       end
+
+      def cip?
+        induction_programme_choice == "core_induction_programme"
```

Comment:
These could be `school_cohort.core_induction_programme?` and `school_cohort.full_induction_programme?` instead?